idd/Energy+.idd.in

```diff
@@ -4862,11 +4862,18 @@ Schedule:File,
        \key Yes
        \key No
        \default No
-  N4 ; \field Minutes per Item
+  N4 , \field Minutes per Item
        \note Must be evenly divisible into 60
        \type integer
        \minimum 1
        \maximum 60
+  A6 ; \field Adjust Schedule for Daylight Savings
```

Comment:
@prsh5175 @nmerket Since this is a new field at the end of the object that is not required and has a default, no transition code is needed. Please change min-fields to 9 at the top of the object.  That will ensure that all defaults get filled in by the input processor and it cause transition to extend the object to show all fields. For the regression example files that use this object, it's sufficient to manually add the new field (and any others that are missing) to help show the settings.

Reply:
@mjwitte I think we do need to transition existing files. Adding this field changes the default behavior of the `Schedule:File` object. To get the existing behavior we need to add the field and set it to "Yes".

Reply:
> Adding this field changes the default behavior of the Schedule:File object. To get the existing behavior we need to add the field and set it to "Yes".

Hmm, I didn't catch that. That's not what we typically want from a new field, but if that's the agreed upon goal here, then ok. This can dangerous, because users with existing schedule file objects will get a different result from a new idf where they just add Schedule:File without thinking about it (so it will default to no) and get a different result from an older file that was transitioned up (which was set to Yes).

So, this raises a new question: If the current behavior is desired, then transition should set this to Yes. But if the current behavior is considered a defect, then perhaps transition should set this to No and put a warning in the transitioned file. Just tossing that out.